docs/WebsocketAPIV5.md

```diff
@@ -0,0 +1,235 @@
+types in the executePreparedStatement request.
+
+| Type | Required Properties | Optional Properties |
+| --- | --- | --- |
+| BOOLEAN | | |
+| CHAR | size | |
+| DATE | | |
+| DECIMAL | precision, scale | |
+| DOUBLE | | |
+| GEOMETRY | | |
+| HASHTYPE | | |
+| INTERVAL DAY TO SECOND | precision, fraction | |
+| INTERVAL YEAR TO MONTH | precision | |
+| TIMESTAMP | | precision, withLocalTimeZone |
+| TIMESTAMP WITH LOCAL TIME ZONE | | precision, withLocalTimeZone |
+| VARCHAR | size | |
+
+
+The following data types and properties are used to specify column
+types in responses from Exasol.
+
+| Type | Properties |
+| --- | --- |
+| BOOLEAN | |
+| CHAR | size, characterSet |
+| DATE | |
+| DECIMAL | precision, scale |
+| DOUBLE | |
+| GEOMETRY | size, srid |
+| HASHTYPE | size |
+| INTERVAL DAY TO SECOND | precision, fraction |
+| INTERVAL YEAR TO MONTH | precision |
+| TIMESTAMP | precision, withLocalTimeZone |
+| TIMESTAMP WITH LOCAL TIME ZONE | precision, withLocalTimeZone |
+| VARCHAR | size, characterSet |
+
+## Compression
+
+The data in the WebSocket data frames may be compressed using zlib. In
+order to enable compression, the client must set the `useCompression`
+field in the login command to true. If compression is enabled during
+login, all messages sent and received after login completion must be
+binary data frames, in which the payload data (i.e., command
+request/response) is zlib-compressed.
+
+## Heartbeat/Feedback messages
+
+The `feedbackInterval` session attribute specifies how often (in seconds)
+unidirectional heartbeat/feedback messages are sent to the client
+during query execution. These messages are sent using Pong WebSocket
+control frames (see RFC 6455), and thus a response is not expected.
+
+The client may send Ping WebSocket control frames (see RFC 6455) to
+Exasol, for example, as client-initiated keepalives. Exasol
+will respond to a Ping frame with a Pong response.
+
+Exasol will not send Ping frames to the client.
+
+## Subconnections
+
+### Introduction
+
+Subconnections are additional connections to Exasol cluster nodes which can be created by the client. There are two main reasons to create and use subconnections.
+1. Read a result set in parallel: [fetch](commands/fetchV1.md) can be called in parallel by each of the subconnections to read a result set.
+2. `INSERT` data in parallel: [executePreparedStatement](commands/executePreparedStatementV1.md) can be called in parallel on each of the subconnections to `INSERT` various data. Please note that the same prepared statement (see [createPreparedStatement](commands/createPreparedStatementV1.md)) must be executed on all subconnections.
+
+ℹ️ Here, parallel refers to the node-wise reading/inserting of data. For example, if there is a subconnection for each node (i.e., the number of subconnections equals the number of cluster nodes), then each subconnection will read/insert data locally from/into its node.
```

Comment:
I think, I needs to be removed in the beginning of the statement. And also in previous versions too.

Reply:
Yeah, they used to be info emojis, but they were somehow removed.